### PR TITLE
Bump test-retry plugin to latest

### PR DESCRIPTION
**Pull request by paplorinc**
See: https://github.com/gradle/test-retry-gradle-plugin/releases/tag/v1.3.1

### CODE REVIEW

**Comment by blindpirate** [comment]
@bot-gradle test this

**Review by blindpirate — APPROVED** [review]
Thanks for doing this!

**Review by snoopcheri — APPROVED** [review]

**Comment by blindpirate** [comment]
@bot-gradle test and merge

File: gradle/verification-metadata.xml [hunk 1]
```diff
@@ -2244,6 +2244,11 @@
             <pgp value="314fe82e5a4c5377bca2edec5208812e1e4a6db0"/>
          </artifact>
       </component>
+      <component group="org.gradle" name="test-retry-gradle-plugin" version="1.3.1">
```
Comment (paplorinc) [inline gradle/verification-metadata.xml:2247]: thanks

**Pull request merged**